vignettes/manual/BuildHector.Rmd

```diff
@@ -51,49 +55,17 @@ Hector's mandatory dependencies are listed in the `DESCRIPTION` file under `Impo
 - [`BH`](https://cran.r-project.org/web/packages/BH/index.html) -- Boost C++ header files
 
 
-# Installing and building - standalone executable
+# Standalone Executable
 
 Hector can also be compiled as a standalone executable.
 Unlike the R package, this method of installation does not automatically pull in dependencies, so you will have to install them manually.
 Fortunately, Hector's only external dependency--Boost--is freely available under a GPL license.
 
 **Boost** is a free, peer-reviewed portable C++ source library, available at http://www.boost.org/.
 Hector primarily uses Boost "header-only" libraries, which do not need to be compiled independently and only need to be extracted and included with the Hector source.
-However, Hector currently _does_ depend on two Boost libraries--`system` and `filesystem`--that require compilation.
+However, Hector currently _does_ depend on two Boost libraries--`system` and `filesystem`--that require compilation. 
 
-## Mac OSX (Xcode)
-
-These directions assume a basic familiarity with Xcode and Mac OS X software installation.
-If you're going to use `make` and not Xcode, see Linux directions below.
-
-* Install [Xcode](https://developer.apple.com/xcode/downloads/) if necessary. Hector has been built and tested with Mac OS 10.8.5 ~~and 10.10 (Yosemite)~~. The project files are for Xcode 5.1.1.
-* Download and install Boost, following instructions above.
-* Download the [Hector zip file](https://github.com/JGCRI/hector/archive/master.zip) or check out the repository using Git.
-* From Xcode, open the project file in `project_files/Xcode/hector.cxodeproj`.
-* Build the project. See below if you encounter errors.
-* Change the current Scheme settings (Scheme->Edit Scheme) and add a command-line argument (*Arguments* tab, e.g. "input/hector_rcp45.ini").
-* Run!
-
-Xcode Build Settings to check/change if you encounter build errors:
-
-* *Architectures-Base SDK*: OS X 10.8 [or OS X version on machine] 
-* *Build Options-Compiler for C/C++/Objective-C*: Default compiler (Apple LLVM 5.1)
-* *Search Paths-Header Search Paths*: "/usr/local/include /usr/local/lib/boost_1_52_0/" [or other header directories]
-* *Search Paths-Library Search Paths*: "/usr/local/lib/" [or other library directories]
-* *LLVM-Language-C++-C++ Standard Library*: libc++ (LLVM C++ standard library with C++11 support)
-* *Linking-Other Linker Flags*: "-lgsl -lgslcblas -lm"
-* *User-Defined*: "GCC_MODEL_TUNING" defined as "G5"
-
-
-## Windows
-
-* Install Visual Studio, if necessary.
-* Download and install *Boost*, following instructions above.
-* Download the [Hector zip file](https://github.com/JGCRI/hector/archive/master.zip) or check out the repository using Git.
-* TODO
-
-
-## Linux
+## Command Line (Linux & Mac OSX)
 
```

Comment:
Does the command line build/run work on windows?

Reply:
Not that I know of.